EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -293,14 +294,40 @@ export default class EventsExpressService {
             }),
         });
 
-        return !res.ok
-            ? {
-                error: {
-                    ErrorCode: res.status,
-                    massage: await res.statusText
-                }
+        if (res.ok) {
+            return await res.json();
+        }
+        else if (res.status === 401) {
+            return this.RefreshHandler(this.getResource, res, url);
+        }
+        else {
+            return this.errorHandler(res);
+        }
+    }
+
+    RefreshHandler = async (callback, res, url, data) => {
```

Comment:
Same here - consider following standard naming convention for UI project.

Reply:
refacored code. Create local func